modules/storages/app/validator/nextcloud_compatible_host_validator.rb

```diff
@@ -27,6 +27,8 @@
 #++
 class NextcloudCompatibleHostValidator < ActiveModel::EachValidator
   MINIMAL_NEXTCLOUD_VERSION = 22
+  TEST_BEARER_TOKEN = "TESTBEARERTOKEN".freeze
+  AUTHORIZATION_HEADER = "Bearer #{TEST_BEARER_TOKEN}".freeze
```

Comment:
`TEST_BEARER_TOKEN` is not reused elsewhere, I'd inline it. Maybe embody `TEST` into the `AUTHORIZATION_HEADER` variable.